ui/src/shared/actions/timeSeries.ts

```diff
@@ -1,12 +1,15 @@
-import {proxy} from 'utils/queryUrlGenerator'
-import {noop} from 'shared/actions/app'
+import {proxy} from 'src/utils/queryUrlGenerator'
+import {noop} from 'src/shared/actions/app'
 import _ from 'lodash'
 
-import {errorThrown} from 'shared/actions/errors'
+import {errorThrown} from 'src/shared/actions/errors'
 
 export const handleLoading = (query, editQueryStatus) => {
-  editQueryStatus(query.id, {loading: true})
+  editQueryStatus(query.id, {
+    loading: true,
+  })
 }
+
 // {results: [{}]}
 export const handleSuccess = (data, query, editQueryStatus) => {
```

Comment:
These are not typed but I think these are also outside the scope of data explorer.